src/sentry/workflow_engine/handlers/condition/issue_priority_deescalating_handler.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from sentry.models.group import GroupStatus
+from sentry.models.groupopenperiod import GroupOpenPeriod
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, WorkflowEventData
+
+
+@condition_handler_registry.register(Condition.ISSUE_PRIORITY_DEESCALATING)
+class IssuePriorityDeescalatingConditionHandler(DataConditionHandler[WorkflowEventData]):
+    group = DataConditionHandler.Group.ACTION_FILTER
+    subgroup = DataConditionHandler.Subgroup.ISSUE_ATTRIBUTES
+
+    @staticmethod
+    def evaluate_value(event_data: WorkflowEventData, comparison: Any) -> bool:
+        group = event_data.event.group
+        if group.status == GroupStatus.RESOLVED:
+            return True
+
+        current_priority = group.priority
+        open_period = GroupOpenPeriod.objects.filter(group=group).order_by("-date_started").first()
```

Comment:
🤔 This feels like a super common operation - i wonder if we should make a helper on the open period to get "latest open period"

Reply:
good call - this and a `get_latest_unresolved_open_period` would be good to have. I can put up an open period utils pr